File: V. PUM/V_PUM.java
import java.util.Scanner;

public class V_PUM {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        int count = 1;

        while (n >= 1) {
            for (int i = 0; i <= 4; i++) {
                if (i == 3) {
                    System.out.print("PUM");
                    count++;
                    break;
                } else {
                    System.out.print(count + " ");
                    count++;
                }
            }
            n--;
            System.out.println();
        }

        sc.close();
    }
}

File: V. PUM/V_PUM.py
n = int(input())
count = 1
while n >= 1:
    for i in range(4):
        if i == 3:
            print("PUM", end="")
            count += 1
            break
        else:
            print(f"{count} ", end="")
            count += 1

    n -= 1
    print()
